itoa.c

```diff
@@ -0,0 +1,81 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+// Converts an integer to a dynamically allocated null-terminated C string
+// Examples:
+//            15 -> ['1', '5', '\0']
+//            -5 -> ['-', '5', '\0']
+//           123 -> ['1', '2', '3', '\0']
+int getNumDigits(int v) {
```

Comment:
To be consistent with `itoa(..)`
```suggestion
int getNumDigits(int v) 
{
```